test/controllers/secure.register.filter.controller.spec.ts

```diff
@@ -87,7 +96,8 @@ describe( "SECURE REGISTER FILTER controller", () => {
   describe("GET tests", () => {
 
     test(`renders the ${config.SECURE_REGISTER_FILTER_PAGE} page`, async () => {
-      mockGetApplicationData.mockReturnValueOnce({});
+      // mockGetApplicationData.mockReturnValueOnce({});
```

Comment:
a few of these commented code can be removed

Reply:
removed